warehouse/organizations/services.py

```diff
@@ -59,6 +66,25 @@ def get_organization_by_name(self, name):
             None if organization_id is None else self.get_organization(organization_id)
         )
 
+    def get_organization_application_by_name(
+        self, name, submitted_by=None, submitted_only=False
+    ):
+        """
+        Return the organization object corresponding with the given organization name,
+        or None if there is no organization with that name.
+        """
+        normalized_name = func.normalize_pep426_name(name)
+        query = self.db.query(OrganizationApplication).filter(
+            OrganizationApplication.normalized_name == normalized_name
+        )
+        if submitted_by is not None:
+            query = query.filter(OrganizationApplication.submitted_by == submitted_by)
+        if submitted_only is True:
+            query = query.filter(
+                OrganizationApplication.is_approved == None  # noqa: E711
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏I know this was only moved, but chance to correct ignores. I _think_ this is the right syntax:

```suggestion
                OrganizationApplication.is_approved.is_(None)
```

Reply:
78376d6ab4f5763c4b86c8f690fbd9e52c658d3e